builtin/logical/pki/path_tidy_test.go

```diff
@@ -1313,3 +1320,39 @@ func waitForTidyToFinish(t *testing.T, client *api.Client, mount string) *api.Se
 	t.Logf("got tidy status: %v", statusResp.Data)
 	return statusResp
 }
+
+func TestRevokedSafetyBufferConfig(t *testing.T) {
+	t.Parallel()
+
+	b, s := CreateBackendWithStorage(t)
+
+	// Verify that the default of revoked_safety_buffer is the default of safety_buffer when neither are set
+	resp, err := CBWrite(b, s, "config/auto-tidy", map[string]interface{}{})
+	resp, err = CBRead(b, s, "config/auto-tidy")
+	requireSuccessNonNilResponse(t, resp, err, "expected to read auto-tidy config")
+	require.Equal(t, resp.Data["safety_buffer"].(int), resp.Data["revoked_safety_buffer"].(int), "expected revoked_safety_buffer to be set to safetyBuffer")
+
+	// Verify that revoked_safety_buffer defaults to safety_buffer when safety_buffer is set
+	safetyBuffer := 3600
```

Comment:
This is the part causing the test failure for reasons specified in https://github.com/openbao/openbao/pull/653#discussion_r1813695576